sharding/test/trivial_integration_test.go

```diff
@@ -142,3 +155,39 @@ func TestErrorsIfShardingKeyChanged(t *testing.T) {
 	assert.NotNil(t, errorHandler.LastError)
 	assert.Equal(t, "sharding key changed from 2 to 1", errorHandler.LastError.Error())
 }
+
+type InsertDDLWriter struct {
+	db *sql.DB
+}
+
+func (this *InsertDDLWriter) Run() {
+	this.db.Exec("ALTER TABLE gftest.table1 MODIFY COLUMN data varchar(255)")
+}
+
+func (this *InsertDDLWriter) Stop() {}
+
+func (this *InsertDDLWriter) Wait() {}
+
+func (this *InsertDDLWriter) SetDB(db *sql.DB) {
+	this.db = db
+}
+
+func TestErrorsIfDDLDetected(t *testing.T) {
```

Comment:
I thought we had a ruby integration test for this? Do we have to use the Golang version instead?